nibabel/analyze.py

```diff
@@ -950,6 +950,18 @@ def from_file_map(klass, file_map, mmap=True):
             `mmap` value of True gives the same behavior as ``mmap='c'``.  If
             image data file cannot be memory-mapped, ignore `mmap` value and
             read array from file.
+        keep_file_open : { 'auto', True, False }, optional, keyword only
```

Comment:
I think you now need `{None, 'auto', True, False}`, and to say that None is the default, and gives the KEEP_FILE_OPEN_DEFAULT value.

Reply:
Likewise for the other docstrings.